introduction.md

```diff
@@ -0,0 +1 @@
+Igurisu-jin no Nick de gozaimasu. Hajimemashite, yoroshiku onegaitashimasu.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.